### PR TITLE
Pulling review average and review count, and reflecting stars and review count on booking component

### PR DESCRIPTION
**Pull request by insivika**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -3,42 +3,61 @@ import DateSelector from './components/DateSelector';
 import Guests from './components/Guests';
 import { library } from '@fortawesome/fontawesome-svg-core';
 import axios from 'axios';
+import faker from 'faker';
 import { StickyContainer, Sticky } from 'react-sticky';
-import { faArrowRight, faIgloo, faPlusCircle, faMinusCircle} from '@fortawesome/free-solid-svg-icons';
+import { faStarHalf, faStar, faArrowRight, faIgloo, faPlusCircle, faMinusCircle} from '@fortawesome/free-solid-svg-icons';
+const moment = require('moment')
```
Comment (kogisu) [inline client/src/App.js]: since you're using imports, i would be consistent and use `import moment from 'moment';`

File: client/src/App.js [hunk 2]
```diff
@@ -3,42 +3,61 @@ import DateSelector from './components/DateSelector';
...
-library.add(faIgloo, faPlusCircle, faMinusCircle, faArrowRight)
+
+let id = '/1';
+if (window.location.pathname !== '/') {
+  id = window.location.pathname;
+}
 
 class App extends React.Component {
 
   state={
```
Comment (kogisu) [inline client/src/App.js]: doesn't state need the `'this'` keyword?

File: client/src/components/guests.js [hunk 3]
```diff
@@ -5,34 +5,95 @@ import './Guests.css'
...
+    children: 0,
+    infants: 0,
   }
 
-  showCard = () => {
+
+updateParentState = () => {
+  const guests = {}
+  
+  guests.adults = this.state.adults;
```
Comment (kogisu) [inline client/src/components/guests.js]: I would simplify lines 18 - 20 with `const newState = Object.assign(this.state, {});`

File: client/src/App.js (hunk 1 above)
Comment (insivika) [reply client/src/App.js]: @kogisu good call, dont rememeber why but I had some issues importing it previously. I re-adjust to use an import stmt and it seems to be working now. Thanks for catching

File: client/src/components/guests.js (hunk 3 above)
Comment (insivika) [reply client/src/components/guests.js]: @kogisu yup, great idea. Changed to reflect this suggestion

**Pull request merged**